docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,15 +1,32 @@
 = Welcome
 
-Welcome to the Apache James Reference Book! This Book is for version {page-version} of James.
+Welcome to the Apache James Reference Documentation! These documents are for version {page-version} of James.
 
-Apache James is the **J**ava **A**pache **M**ail **E**nterprise **S**erver. It boasts of a modular architecture based on a rich set of components that run on the JVM. With James, you can create your own custom enterprise email solution.
+Apache James is a flexible email platform that provides both out-of-the-box mail servers that you can start
+using immediately as well as a rich set of components that you can assemble to build your own solution.
+James runs on the JVM and is highly customizable.
 
 This Reference Book is divided into the following Parts:
 
-* xref:main:user:index.adoc[User Manual]
-* xref:main:ops:index.adoc[Operator Manual]
-* xref:main:admin:index.adoc[Administrator Guide]
-* xref:main:dev:index.adoc[Developer Resources]
-* xref:main:doc:index.adoc[About this documentation]
+* xref:main:concepts:index.adoc[James Core Concepts]
+**    These are the core concepts that describe what James is all about.
+    Start here if you want to learn more about what James can do
+    and how it is designed.
+* xref:main:servers:index.adoc[James Servers]
+**    We provide a few out-of-the-box servers that you can 
+    choose from depending on your needs. Start here if you just
+    want to get up and running quickly with a working mail server.
+* xref:main:toolkit:index.adoc[James Toolkit]
+**    Describes how you can assemble your own specialized server by 
+    using the various James components as a toolkit.
+    This section is aimed at experienced James users.
+* xref:main:customization:index.adoc[Customizing James]
```

Comment:
I would call this section `Customizing James servers` and would move it above the `toolkit` section

Reply:
@chibenwa, to help me understand, could you please explain to me what, in your mind, is the difference between "customization" and "assembly"?

They both are a way of "changing" James to accomplish a specific agenda.

Customization is a way of "changing" James from the "outside". Assembly is a way of using existing parts to put them together to form a certain solution.

Customization can work on any assembly. (Is that true??)

Why would somebody want to do one over the other? Why do you suggest that customization be first on the list?

Reply:
Customization is available on pre-built products as far as assembled products. It mainly some XML configuration. In some cases in can require to write one or 2 java classes (mailet / matcher) and put them in a directory to accomplish a particular customized workflow.

Assembly need to create a new Guice product, choose a particular set en Guice modules, and build this particular product. It's more for developpers than for experienced users (note that it was not the case with Spring, were with some XML magic you could reach the same objective).

Reply:
Switched the order of the sections as suggested and updated the contents consequently.